content/docs/atom-guide/dev-guide/shaders/azsl/srg-semantics.md

```diff
@@ -1,36 +1,38 @@
 ---
 linktitle: SRG Semantics
-title: AZSL, Shader Resource Group Semantics (ShaderResourceGroupSemantic).
-description: Learn about AZSL Shader Resource Group Semantics in the Atom Renderer.
+title: Shader Resource Group Semantics (ShaderResourceGroupSemantic).
+description: Learn about Amazon Shading Language (AZSL) shader resource group semantics (SRG) for Atom Renderer.
 weight: 100
 ---
 
 The `ShaderResourceGroupSemantic`, SRG Semantic, for short, defines the order in which descriptor bindings and descriptor sets (descriptor sets are also known as register spaces in HLSL) are emitted for each `ShaderResourceGroup`.  
-  
+```cpp
     ShaderResourceGroupSemantic <name>
     {
         FrequencyId = <0-based index>;
         [ShaderVariantFallback = <number of bits>;] // (Optional)
     }
-  
-Conceptually, an SRG Semantic relates to the frequency of change of SRG data. In particular, the render pipeline has a predefined set of SRG Semantics in "&lt;Engine Root&gt;/Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/SrgSemantics.azsli".  
-https://github.com/o3de/o3de/blob/main/Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/SrgSemantics.azsli  
-  
-* `SRG_PerDraw`: Data changes per draw packet. Highest frequency.
-* `SRG_PerObject`: Data changes per object.
-* `SRG_PerMaterial`: Data changes per material.
-* `SRG_PerSubPass`: Data changes per sub-pass.
-* `SRG_PerPass`: Data changes per pass.
-* `SRG_PerPass_WithFallback`: Same as SRG_PerPass, but used for cases where the Pass SRG defines the Fallback key.
-* `SRG_PerView`: Data changes per viewport, because each viewport has its own camera.
-* `SRG_PerScene`: Data changes per scene. Lowest frequency.
-* `SRG_RayTracingGlobal`: Reserved for ray tracing shaders.
-* `SRG_RayTracingLocal_[1..7]`: Reserved for ray tracing shaders.
-  
-## FrequencyId: Defining the order of emission and register space utilization.
-With the value of `FrequencyId` we can define in what order we utilize the register space when compiling SRGs.  
+```
+An SRG semantic relates to the frequency of change of SRG data. The render pipeline has a predefined set of SRG semantics in [`<Engine Root>/Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/SrgSemantics.azsli`](https://github.com/o3de/o3de/blob/main/Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/SrgSemantics.azsli) 
```

Comment:
```suggestion
An SRG Semantic relates to the frequency of change of SRG data. The render pipeline has a predefined set of SRG semantics in [`<Engine Root>/Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/SrgSemantics.azsli`](https://github.com/o3de/o3de/blob/main/Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/SrgSemantics.azsli) 
```
* Be consistent with terminology. Should it be "SRG semantic" or "SRG Semantic" with a capital "S"?

Reply:
Fixed.